### PR TITLE
Feat/275

### PR DESCRIPTION
**Pull request by fn-faisal**
https://github.com/parse-community/parse-dashboard/issues/275

Array of pointers. 

`[
  {
    "__type": "Pointer",
    "className": "Test",
    "objectId": "YwW26oAyFB"
  },
  {
    "__type": "Pointer",
    "className": "Test",
    "objectId": "VyvVSg0qDn"
  },
  {
    "__type": "Pointer",
    "className": "Test",
    "objectId": "YwW26oAyFB"
  }
]`

Screenshot 2021-06-14 at 2 46 17 PM
Screenshot 2021-06-14 at 2 46 26 PM
Screenshot 2021-06-14 at 2 46 43 PM

### CODE REVIEW

File: Parse-Dashboard/parse-dashboard-config.json [hunk 1]
```diff
@@ -2,15 +2,13 @@
   "apps": [
     {
       "serverURL": "http://localhost:1338/parse",
-      "appId": "APPLICATION_ID",
-      "masterKey": "MASTER_KEY",
-      "appName": "Countries States and Cities",
+      "appId": "hello",
```
Comment (davimacedo) [inline Parse-Dashboard/parse-dashboard-config.json:5]: @fn-faisal could you please revert the changes in this file?

Comment (fn-faisal) [reply Parse-Dashboard/parse-dashboard-config.json]: I have reverted the config to match the config in master b4a branch.
